skimage/restoration/_denoise.py

```diff
@@ -134,7 +134,7 @@ def denoise_tv_bregman(image, weight, max_iter=100, eps=1e-3, isotropic=True):
 
     Returns
     -------
-    u : ndarray
+    out : ndarray
         Denoised image.
```

Comment:
I don't think that's very clear... I would suggest that we keep `out` or `denoised` (both are used in the file) and replace u in the equation with that name. What do you think?